homeassistant/components/tplink_omada/quality_scale.yaml

```diff
@@ -0,0 +1,74 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: done
+    comment: Service data APIs are polled every 5 minutes
+  brands: done
+  common-modules:
+    status: done
+    comment: Centralised coordinator, controller and base entity classes are used.
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency:
+    status: done
+    comment: API dependency published on PyPI, MIT licensed.
+  docs-actions:
+    status: done
+    comment: "PR #41801 adds action documentation."
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions:
+    status: done
+    comment: "PR #41801 adds standard removal documentation."
+  entity-event-setup:
+    status: done
+    comment: Integration does not subscribe to events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry:
+    status: done
+    comment: Omada Site unique ID checked during config flow.
+
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: done
+  docs-configuration-parameters: todo
+  docs-installation-parameters: todo
+  entity-unavailable: todo
```

Comment:
Is there more that you want to do on this (as your coordinator has this)? (Could apply to log-when-unavailable as well)